internal/resources/grafana/resource_alerting_contact_point.go

```diff
@@ -61,12 +61,19 @@ This resource requires Grafana 9.1.0 or later.
 		},
 	}
 
+	// Build list of available notifier fields, at least one has to be specified
+	notifierFields := []string{}
+	for _, n := range notifiers {
+		notifierFields = append(notifierFields, n.meta().field)
+	}
```

Comment:
**Feel free to ignore this**

Low hanging perf optimization:
```suggestion
	notifierFields := make([]string{}, len(notifiers))
	for i, n := range notifiers {
		notifierFields[i] = n.meta().field
	}
```

I suppose we could even move this out into an `init` function so if you use this resource more than once we don't need to recompute this slice over and over.